g2p_registry_base/models/reg_id.py

```diff
@@ -16,6 +16,8 @@ class G2PRegistrantID(models.Model):
     )
     id_type = fields.Many2one("g2p.id.type", "ID Type", required=True)
     value = fields.Char(size=100)
+    status = fields.Char()
+    error = fields.Char()
```

Comment:
What could be in this as it is just to store the ID document number?

Reply:
There is no expected syntax for these values as of now, they are just string fields. Example;
status: "ERR"
error: "First Name doesn't match"
But maybe OpenG2P could generalize a syntax for this in future.

Reply:
For status at least, we should standardize it with a selection instead. Letting the user set anything will make it hard to have different modules interpret its value.
For the error field I'm good with free form.